Makefile

```diff
@@ -1,6 +1,16 @@
 # Config
 WORKERS = 4
 SHELL = /bin/bash
+MANAGE := python ./manage.py
+HONCHO_MANAGE := honcho run $(MANAGE)
+
+.PHONY: install_system_dependencies
```

Comment:
@brousch Good point, all of the target are actually phony targets. Could you add it on the other ones too, for consistency?